libsrc/apple2/ser/a2.gs.s

```diff
@@ -73,86 +73,90 @@ CurChanIrqFlags:.byte   INTR_PENDING_RX_EXT_B
 
 SerFlagOrig:    .byte   $00
 
-; Tables used to translate RS232 params into register values
+; Tables used to translate cc65 RS232 params into register values
 ; (Ref page 5-18 and 5-19)
 BaudLowTable:
```

Comment:
Maybe the line feed here wants to go away so that the label/symbol is on the same line as the first entry - basically like the items  above. The same idea applies to all tables below as well.

Reply:
Damn, I wanted to do that and forgot 😅
Will do!

Reply:
Done with the exception of BaudTable, where there is an extra information about bit7 which I thought was uglier if above. What do you think?

Reply:
I agree with you that it's best the way it is now 👍🏻